README.md

```diff
@@ -2,6 +2,17 @@
 
 This is a IPAM provider for Cluster API that manages pools of IP addresses using Kubernetes resources. It serves as a reference implementation for IPAM providers, but can also be used as a simple replacement for DHCP.
 
+## Cluster API setup via clusterctl
```

Comment:
Can we remove "Cluster API" here? A bit misleading since we only want to setup the provider instead of the entirety Cluster API 😄